### ISSUE TITLE
[v1] [Feature] QTabs - option to display arrows on mobile?

### ISSUE THREAD

**Issue by mpacary** [quasarframework/quasar#4019]
According to the [**QTabs** doc page](https://v1.quasar-framework.org/vue-components/tabs), if the **QTabs** content is larger than available space, arrows/chevrons are not displayed on **QTabs** side when on mobile.

> QTabs can be scrolled horizontally when the width is longer than the container width. Adjust your browser accordingly to see this in action. On a desktop you will see chevons on either side that can be clicked. On a mobile, you can pan the tabs with your finger.

It turns to be a problem, because sometimes the right edge of **QTabs** is between two tabs (one tab is displayed, the next isn’t)

Therefore, a newcoming user has no way to tell if there is more tabs or not.

Would it be possible to have an option allowing to display a right arrow showing clearly to the user that more tabs are available?

![image](https://user-images.githubusercontent.com/705663/57070311-4c934880-6cd7-11e9-944d-86e54ba97a85.png)

**Comment by AjiTae** [comment]
**workaround**

before all:
```javascript
import {debounce, QTabs} from 'quasar';

QTabs.mixin({
    mounted() {
        if (this.$refs.content) {
            this.$refs.content.addEventListener('scroll', debounce(this.__updateArrows, 150));
        }
    }
});
```

css:
```stylus
body.mobile
  .q-tabs__arrow
    display: inline-flex
  .q-tabs--not-scrollable .q-tabs__arrow
    display: none
```

**Comment by turigeza** [comment]
@AjiTae Does it still work for you ? For me it doesn't ... mind I don't even see the DOM element `.q-tabs__arrow`. They don't seem to render on a mobile.

**Comment by AjiTae** [comment]
Yep, [this](https://github.com/quasarframework/quasar/commit/4d560cf2aef039d50697973f5dcaa820fd84a73b) commit brokes it.
We need official support for such behavior because monkey-patching is as unreliable as always.

**Comment by jblyberg** [comment]
Has this change been incorporated into release yet? Arrows are not showing up for me on mobile.

**Comment by cristianomoraes** [comment]
This feature should be incorporated into Quasar release. Should this ticket get a `feature request` label?

**Comment by nseb** [comment]
+ same problem , no arrows ;-(

**Comment by TG4LAaron** [comment]
Any updates on this issue?

**Comment by turigeza** [comment]
Apparently this is how it should work ... 
https://forum.quasar-framework.org/topic/6102/no-arrows-in-q-tab-in-mobile-device/2

I don't get it though and from my experience people are missing the arrows.

**Comment by webnoob** [comment]
Will be available in 1.12.9
